pkg/client/standalone.go

```diff
@@ -79,30 +86,20 @@ func (c *StandaloneClient) IngestPaths(ctx context.Context, datasetID string, op
 	}
 
 	ingestFile := func(path string, extraMetadata map[string]any) error {
-		// Gather metadata
-		finfo, err := os.Stat(path)
-		if err != nil {
-			return fmt.Errorf("failed to stat file %s: %w", path, err)
-		}
-
-		abspath, err := filepath.Abs(path)
-		if err != nil {
-			return fmt.Errorf("failed to get absolute path for %s: %w", path, err)
-		}
 
-		file, err := os.ReadFile(path)
+		file, err := c.GPTScript.ReadFileInWorkspace(ctx, path)
 		if err != nil {
-			return fmt.Errorf("failed to open file %s: %w", path, err)
+			return fmt.Errorf("failed to read file %q: %w", path, err)
 		}
 
 		filename := filepath.Base(path)
 
 		iopts := datastore.IngestOpts{
 			FileMetadata: &index.FileMetadata{
 				Name:         filepath.Base(path),
-				AbsolutePath: abspath,
-				Size:         finfo.Size(),
-				ModifiedAt:   finfo.ModTime(),
+				AbsolutePath: path,
+				Size:         int64(len(file)), // TODO: get from file metadata once the workspace SDK supports it
+				ModifiedAt:   time.Now(),       // TODO: get from file metadata once the workspace SDK supports it
```

Comment:
Does this mean we will always re-ingest since the modified time will always be different?